SpywareFilter/sections/specific.txt

```diff
@@ -80,6 +80,7 @@ $cookie=_awl;maxAge=30,domain=nypost.com|al.com|9news.com
 !
 ! NOTE: Specific rules
 !
+||yakutia.info^$third-party
```

Comment:
What the purpose?

Reply:
@Alex-302 Could you please take a look at the question from - https://github.com/AdguardTeam/AdguardFilters/pull/201119#pullrequestreview-2694108636?

Reply:
One moment.

Reply:
I think better to add a new domain to `||relap.io^$domain=`
In any case `||yakutia.info^$third-party` is weird. Maybe just misunderstanding?

Reply:
@Nirineya Please take a look ⬆️

Reply:
`||yakutia.info^$third-party` has been replaced by `yakutia.info##.popular:has(> .js-relap-anchor)`.
I just wonder if it should be added to `AdGuard Tracking Protection filter` or to `AdGuard Russian filter` + `||relap.io^$domain=`

Reply:
If "it" = `yakutia.info##.popular:has(> .js-relap-anchor)` - I'd add it to both filters, and update `||relap.io^$domain=`:)

Reply:
But current rules are also ok and you can merge it.

Reply:
Okay.